crml/cennzx-spot/rpc/runtime-api/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Parity Technologies (UK) Ltd. and Centrality Investments Ltd.
+// Copyright 2018-2020 Parity Technologies (UK) Ltd. and Centrality Investments Ltd.
```

Comment:
Cennzx-spot is not part of substrate, so Parity can probably be removed here

Reply:
changed all files in the spot exchange into centrality's copyright text